.github/workflows/screener-run.yml

```
@@ -101,6 +141,7 @@ jobs:
           workflow_conclusion: success
           name: screener-artifact
           path: apps/vr-tests/dist/storybook
+        if: ${{ env.IS_ARTIFACT_PRESENT == 'false' }}
```

```suggestion
        if: ${{ env.IS_ARTIFACT_PRESENT == 'true' }}
```

Should it be `true` instead?

---

yes, a logic error occurred after renaming the variable, but I will make sure it's fixed